app/views/shared/_meta.html.haml

```diff
@@ -0,0 +1,5 @@
+%meta{bane: 'title', content: "#{content_for :page_title_prefix}".strip}/
```

Comment:
Is 'bane' a typo for 'name' or is this a meta tag I've never heard of?

Reply:
My right index finger was off by one 🤣

Reply:
![bane_tdkr3](https://user-images.githubusercontent.com/1089521/48079961-7403cf80-e1e4-11e8-8ed0-af9189ed1848.jpg)

Reply:
👹